api/middleware/authRequired.js

```diff
@@ -27,15 +35,37 @@ const authRequired = async (req, res, next) => {
       });
     }
 
+    /**
+     *  @author Khaleel Musleh
+     * @abstract Changed from Okta to Auth0
+     * Substituted: 
+    //const oktaData = await oktaJwtVerifier.verify(
+    //   idToken,
+    //   AuthVerifierConfig.audience
+    // );
+
```

Comment:
If you decide all the comments are necessary, take advantage of your whitespace and parallel structure. Your "TO:" block is clearly defined, but "Substituted" does not match in styling, spacing, or vernacular. Try "FROM:" with "TO:", or "REPLACED:" and "WITH:".